src/test/clojure/clara/test_accumulators.clj

```diff
@@ -59,10 +59,15 @@
         session (-> empty-session
                     (insert (->Temperature 30 "MCI"))
                     (insert (->Temperature 10 "MCI"))
-                    (insert (->Temperature 80 "MCI")))
+                    (insert (->Temperature 80 "MCI"))
+                    fire-rules)
 
-        min-retracted (retract session (->Temperature 10 "MCI"))
-        max-retracted (retract session (->Temperature 80 "MCI"))]
+        min-retracted (-> session
```

Comment:
there are no assertions in this test on this
assuming we'd want an assertion on min-retracted